include/wx/msw/setup0.h

```diff
@@ -1432,9 +1444,9 @@
 // Should wxDC provide SetTransformMatrix() and related methods?
 //
 // Default is 1 but can be set to 0 if this functionality is not used. Notice
-// that currently wxMSW, wxGTK3 support this for wxDC and all platforms support
-// this for wxGCDC so setting this to 0 doesn't change much if neither of these
-// is used (although it will still save a few bytes probably).
+// that currently only wxMSW supports this so setting this to 0 doesn't change
+// much for non-MSW platforms (although it will still save a few bytes
+// probably).
```

Comment:
@a-wi I've replaced this modification with 4cc45797a120d3ac27e5bc2c417faa03da0d016a which ensures that the changes of 49000defcfb11b409d8935126981b14169ee62a3 are not lost during regeneration of the files with `build/update-setup-h`. Does this address your comment?